dispersydatabase.py

```diff
@@ -84,255 +89,13 @@ def check_database(self, database_version):
-
-            # upgrade from version 10 to version 11
-            if database_version < 11:
-                self._logger.debug("upgrade database %d -> %d", database_version, 11)
-                # unfortunately the default SCHEMA did not contain
-                # sync_global_time_undone_meta_message_index but was still using
-                # sync_meta_message_global_time_index in database version 10
-                self.executescript(u"""
-DROP INDEX IF EXISTS sync_meta_message_global_time_index;
-DROP INDEX IF EXISTS sync_global_time_undone_meta_message_index;
-CREATE INDEX sync_meta_message_undone_global_time_index ON sync(meta_message, undone, global_time);
-UPDATE option SET value = '11' WHERE key = 'database_version';
-""")
-                self.commit()
-                self._logger.debug("upgrade database %d -> %d (done)", database_version, 11)
-
-            # upgrade from version 11 to version 12
-            if database_version < 12:
-                # according to the profiler the dispersy/member.py:201(has_identity) has a
-                # disproportionally long runtime.  this is easily improved using the below index.
-                self._logger.debug("upgrade database %d -> %d", database_version, 12)
-                self.executescript(u"""
-CREATE INDEX sync_meta_message_member ON sync(meta_message, member);
-UPDATE option SET value = '12' WHERE key = 'database_version';
-""")
-                self.commit()
-                self._logger.debug("upgrade database %d -> %d (done)", database_version, 12)
-
-            # upgrade from version 12 to version 13
-            if database_version < 13:
-                self._logger.debug("upgrade database %d -> %d", database_version, 13)
-                # reference_member_sync is a very generic but also expensive way to store
-                # multi-sighned messages.  by simplifying the milti-sign into purely double-sign we
-                # can use a less expensive (in terms of query time) table.  note: we simply drop the
-                # table, we assume that there is no data in there since no release has been made
-                # that uses the multi-sign feature
-                self.executescript(u"""
-DROP TABLE reference_member_sync;
-CREATE TABLE double_signed_sync(
- sync INTEGER REFERENCES sync(id),
- member1 INTEGER REFERENCES member(id),
- member2 INTEGER REFERENCES member(id));
-CREATE INDEX double_signed_sync_index_0 ON double_signed_sync(member1, member2);
-UPDATE option SET value = '13' WHERE key = 'database_version';
-""")
-                self.commit()
-                self._logger.debug("upgrade database %d -> %d (done)", database_version, 13)
+            # Check if the version is not higher than our current known version number.
+            if database_version > LATEST_VERSION:
+                raise DatabaseVersionTooHighError(u"Your database version exceeds the latest version.")
```

Comment:
"This database was created with a more recent dispersy version" or something along the line